ivy/functional/backends/numpy/linear_algebra.py

```diff
@@ -388,10 +388,21 @@ def vector_norm(
     elif ord == -inf:
         return np.min(abs_x, axis=axis, keepdims=keepdims, out=out)
     else:
-        return (
+        # There is a rounding error whenever the input is a 0-dim
+        # The solution at the moment is to convert the 0-dim array to 1-dim
+        # and then convert it back to 0-dim in case of keepdims=True
+        if x.ndim == 0:
+            abs_x = np.expand_dims(abs_x, axis=0)
+
+        res = (
             np.sum(abs_x**ord, axis=axis, keepdims=keepdims) ** (1.0 / ord)
         ).astype(abs_x.dtype)
 
+        if keepdims and x.ndim == 0:
+            res = res.reshape(x.shape)
```

Comment:
I think using `np.squeeze` here would make it easier to understand for someone looking at the code later. Just a minor change, rest looks good to me. Feel free to merge the PR, thanks @fabriziocacicia 😄

Reply:
🫡